README.md

```diff
@@ -63,6 +63,69 @@ See [src/test/resources/empty-project](src/test/resources/simple-project) for an
 
 You can configure the plugin using the `<configuration>` element.
 
+#### Traced Directories
+
+By default the OFT plugin imports requirements from the following directories:
+
+* The `doc` sub-directory of the module that includes the plugin if it exists
+* For each Maven module in the project if they exist:
+  * Compile source roots (default: `src/main/java`)
+  * Resources (default: `src/main/resources`)
+  * Test compile source roots (default: `src/test/java`)
+  * Test resources (default: `src/test/resources`)
+
+##### Adding Custom Source Directories
+
+You can add additional custom source directories using the [Build Helper Maven Plugin](https://www.mojohaus.org/build-helper-maven-plugin/):
+
+```xml
+<plugin>
+    <groupId>org.codehaus.mojo</groupId>
+    <artifactId>build-helper-maven-plugin</artifactId>
+    <version>3.5.0</version>
+    <executions>
+        <execution>
+            <id>add-source</id>
+            <phase>generate-sources</phase>
```

Comment:
It might be a good idea to mention that the `phase` here has nothing to do with in which phase OFT does its job, rather it defines in which phase the directory is added to the list of known source directories by the `build-helper-maven-plugin`.

Some people who are not Maven-savvy might be confused by this snippet.